tests/unit/.eslintrc.json

```diff
@@ -4,6 +4,7 @@
     "jest": true
   },
   "globals": {
+    "global": true,
```

Comment:
What does this do?

Reply:
In Node.js, the global keyword is used to access global objects.

Reply:
Oh, so this is telling eslint to not complain when it sees the `global` global?

Reply:
Yes